My Portflio

tingamapuro04 opened the pull request:
Added
1. index.html
2. styles.css
3. .gitignore
4. linter.yml

tingamapuro04 committed:
linter configuration

tingamapuro04 committed:
ignore file

tingamapuro04 committed:
Addition of image files

tingamapuro04 committed:
html file pre-written in vscode

tingamapuro04 committed:
added style file

tingamapuro04 committed:
readme file

tingamapuro04 committed:
json files vscode

tingamapuro04 committed:
linter errors

danmainah reviewed (changes requested):
Hi @tingamapuro04 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that one linter is not passing as seen below,
![image](https://user-images.githubusercontent.com/61408860/164726323-9edc12db-1a75-43b3-a2d8-afed20d6d7a9.png)


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

tingamapuro04 committed:
css linter correction

tingamapuro04 committed:
css update

tingamapuro04 committed:
css update

tingamapuro04 committed:
errors

tingamapuro04 committed:
errors

tingamapuro04 committed:
update

tingamapuro04 committed:
update

tingamapuro04 committed:
update

tingamapuro04 committed:
update

tingamapuro04 committed:
linter corrections

tingamapuro04 committed:
changes

tingamapuro04 committed:
changes

tingamapuro04 committed:
changes

tingamapuro04 committed:
changes

tingamapuro04 committed:
changes

tingamapuro04 committed:
removed background color

bimbolabuari commented on index.html:
```diff
@@ -0,0 +1,51 @@
+            <li><a href="#">Contact</a></li>
+        </ul>
+
+        <div class="free">
+
+        </div>
```
Kindly remove all redundant code to make your code cleaner and professional 👍

bimbolabuari commented on styles.css:
```diff
@@ -0,0 +1,140 @@
+  img {
+    margin: 5px;
+  }
+}
```
Let's style the web app according to the Figma template you choose(Mobile version)
![image](https://user-images.githubusercontent.com/66112760/164886564-a9696518-aa6d-4143-8827-d9d379b2a7b6.png)

tingamapuro04 replied:
@bimbolabuari I am not sure on how to set up the mobile version because i thought using media queries would do that. If there is another way please offer a guide

bimbolabuari commented on index.html:
```diff
@@ -0,0 +1,51 @@
+
+<body>
+    <div class="Mobile">
+        <ul class="header">
+            <li class="first"><a href="#">My Logo</a></li>
+            <li><a href="#">Portflio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
```
Since you are still working on the mobile version, I think we need the hamburger icon instead of listing out the nav link as in the desktop version. 👍

bimbolabuari commented on index.html:
```diff
@@ -0,0 +1,51 @@
+
+
+        <div class="title">
+            <p>I’m Mariane
```
As a best practice in semantic HTML, I suggest you use `h1` tag for this as it's the main focus of the webpage. 👍

bimbolabuari commented on index.html:
```diff
@@ -0,0 +1,51 @@
+        <div class="social">
+            <p>LET’S CONNECT</p>
+        </div>
+        <div class="icon">
+            <img src="/photos/twitter.jpg" alt="twitter">
+            <img src="/photos/medium.jpg" alt="mail">
+            <img src="./photos/github.jpg" alt="github">
+            <img src="/photos/angellist.jpg" alt="angellist">
+        </div>
```
Let's use `ul` tag to list out items and `li` tag for individual items 👍

bimbolabuari reviewed (changes requested):
Hi @tingamapuro04, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Correct Github flow
- Use of flexbox 
- No linter error

## Required Changes ♻️
- [ ] Kindly add a more descriptive Pull Request 👍 
- [ ] Check the comments under the review.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- Lets use a more descriptive branch name next time 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

tingamapuro04 committed:
setting up flexboxes

tingamapuro04 committed:
updated readme

tingamapuro04 committed:
readme updates

tingamapuro04 committed:
Figma styles

tingamapuro04 committed:
Removing unnecessary spaces as suggested by linters

tingamapuro04 committed:
Correction of an image element path

tingamapuro04 committed:
Style changes as suggested by linter

tingamapuro04 committed:
Font-family changes as per linter

tingamapuro04 committed:
Various changes suggested by linter

tingamapuro04 committed:
linter suggested changes

tingamapuro04 committed:
Adding more information on rhe README.md

tingamapuro04 commented:
I am working on the readme branch and from the previous pull request i have had an overhaul of the whole page since the last one wasn't giving the correct display. Now this follows the requirements of Figma template. I have files such as
1.index.html
2.nina.css
3. .gitignore where i have added node_modules folder
4. README.md file

omar-muhamad commented on index.html:
```diff
@@ -0,0 +1,47 @@
+
+<body>
+    <div class="container">
+        <div class="header hoverable">
+            <h1>My Logo</h1>
+            <img src="/photos/Enabled.png" alt="Hamburger Icon">
+        </div>
```
- [ ] Please replace this `<div>` tag with `<header>` tag for semantic purposes.
- [ ] kindly replace the `<h1>` tag with a `<a>` tag for semantic purposes because when you click on your site logo it supposes to take you to the home page.
- [ ] Kindly wrap the hamburger icon `<img>` tag with a `nav` tag because semantically your navigation button and links suppose to be in a `<nav>` tag.

omar-muhamad commented on index.html:
```diff
@@ -0,0 +1,47 @@
+        </div>
+    
+    
+        <div class="Main">
+            <div class="title hoverable">
+                <h1>I’m Mariane Glad to see you!</h1>
+            </div>
+            <div class="text hoverable">
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+                experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
+            <div class="social hoverable">
+                <h4 class="hoverable">LET’S CONNECT</h4>
+                <ul class="media">
+                    <li><img class="twi" src="/photos/twitter.jpg" alt="Twitter Icon"></li>
+                    <li><img class="lin" src="/photos/Linkedin-icon.jpg" alt="Linkedin Icon"></li>
+                    <li><img class="mil" src="/photos/medium.jpg" alt="Mil Icon"></li>
+                    <li><img class="git" src="/photos/github.jpg" alt="github Icon"></li>
+                    <li><img class="angel" src="/photos/angellist.jpg" alt="Angellist Icon"></li>
+                    
+                </ul>
+            </div>
```
- [ ] For semantic reasons Please wrap this whole section with `<section>` tag instead of `<div>` tag.
- [ ] nice work adding the social media icons inside an unordered list but kindly wrap each `<img>` tag with `<a>` tag and put your social media links because at the end this will be your portfolio.
for example, make it like that:
`<li><a href="your link"><img class="twi" src="/photos/twitter.jpg" alt="Twitter Icon"></a></li>`

omar-muhamad commented on nina.css:
```diff
@@ -0,0 +1,132 @@
+.container {
+  background: url(/photos/header-shapes\ mobile.png)  no-repeat;
+  background-size: cover;
```
- [ ] After you implement the HTML Changes please add this image to the `<section>` tag you added instead of adding it to the whole div containing the header and the headline section.
and you can use this [link](https://www.w3schools.com/cssref/pr_background-position.asp) to help you position it right and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/165501675-df3cb4a2-0040-4342-a938-ef4ba379bbba.png)  | ![image](https://user-images.githubusercontent.com/68796232/165501350-3ee73d89-cfcc-4860-8f55-7388791878ec.png)  |

omar-muhamad commented on nina.css:
```diff
@@ -0,0 +1,132 @@
+  line-height: 1.4em;
+}
+
+.title h1 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
```
- [ ] Please add a margin-top value to this title to match the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/165502401-a296b33c-c26d-4365-a806-fd1a23c9209e.png)  | ![image](https://user-images.githubusercontent.com/68796232/165502686-d7064450-8ca8-4cda-a6c6-7cbeb47d0d6d.png) |

omar-muhamad reviewed (changes requested):
# Changes Required ♻️
### Hi @tingamapuro04,

Nice job, you are almost there.

![](https://camo.githubusercontent.com/56bcd754643682fcc078b66eaa7278dead6c9c5013595c875b4ddcbe0f0ef0aa/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f78554f786668365a4d373565664d3342716f2f67697068792d646f776e73697a65642d6d656469756d2e676966)

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

tingamapuro04 committed:
Added <heade>, <nav> and <a> tags to the class= header hoverable

tingamapuro04 committed:
Added achor tags to the image tags in  the class=social hoverable

tingamapuro04 committed:
Putting the background image in the section area

tingamapuro04 committed:
Adding the height of the section by 10px

tingamapuro04 committed:
Linter correction on index.html

tingamapuro04 committed:
Removed target attribute from links

tingamapuro04 committed:
Linter corrections

tingamapuro04 commented:
I was working on the readme branch where I have used Flexbox generally and I have files such as nina.css, index.html, a readme file, gitignore. I have made changes such as 
1. I have replaced a <div> with a <header>
2.  I have replaced a <h1> with <a> because it is a link
3.  I have wrapped the Humburger icon with a <nav> tag
4.  I have wrapped the whole of it in a <section> tag instead of a <div>
5.  I have added anchor tags <a> to the social media icons with links to various social media accounts
6.  I have added a background image to the <section> only rather than the whole body
7.  I have added a margin top to the header part.

vic778 reviewed (approved):
Hi @tingamapuro04  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![Charlie Brown](https://media.giphy.com/media/ZdO4NenDbsQNwUwKDB/giphy.gif)

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
